tests/database/migrations/2024_06_29_000000_create_episodes_table.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+final class CreateTestbenchNullConnectionsTable extends Migration
```

Comment:
Wrong name

Reply:
fixed it
`final class CreateTestbenchPodcastsTable extends Migration`